features/lorawan/lorastack/mac/LoRaMac.cpp

```diff
@@ -125,851 +119,698 @@ LoRaMac::LoRaMac()
 
     _params.sys_params.adr_on = false;
     _params.sys_params.max_duty_cycle = 0;
-
-    mac_primitives = NULL;
-    ev_queue = NULL;
 }
```

Comment:
Add initialization of _mcps_indication, _mcps_confirmation, _mlme_indication and _mlme_confirmation